ngshare/ngshare.py

```diff
@@ -886,6 +886,15 @@ def get(self):
         self.write('\n' + self.request.uri + '\n' + self.request.path + '\n')
 
 
+class HealthCheckHandler(RequestHandler):
+    'Health check handler on /healthz, for k8s'
+
+    def get(self):
+        "Returns a 200 to indicate we're alive"
+        self.write(json.dumps({'success': True}))
```

Comment:
You can also use `self.json_success()`

Reply:
I didn't inherit from `MyRequestHandler`, only `RequestHandler`, since I don't want the authentication / database access stuff. That's why I don't have access to `self.json_success()`.